ts/config.ts

```diff
@@ -71,6 +71,9 @@ export const cgnTestOverlay: boolean = Config.CGN_TEST_OVERLAY === "YES";
 export const euCovidCertificateEnabled: boolean =
   Config.EU_COVID_CERT_ENABLED === "YES";
 
+// SiciliaVola Feature Flag
+export const svEnabled: boolean = Config.CGN_ENABLED === "YES";
```

Comment:
Probably I'm missing some context here: I'd expect this statement to be `Config.SICILIAVOLA_ENABLED === "YES";` instead.

Reply:
Yes thank you, copy & paste error! 
Fixed in  [cf30101](https://github.com/pagopa/io-app/pull/3124/commits/cf301018ceb22637d88a30e7d6c7d5dc92cc56bd)